packages/commonwealth/client/scripts/views/components/Mava/Mava.tsx

```diff
@@ -0,0 +1,49 @@
+import { memo, useCallback, useEffect, useRef } from 'react';
+import app from 'state';
+import { isWindowSmall } from 'views/components/component_kit/helpers';
+import './Mava.scss';
+
+// eslint-disable-next-line no-var, @typescript-eslint/no-explicit-any
+declare var window: any;
+
+const mobile = isWindowSmall(window.innerWidth);
```

Comment:
I think this should be inside the component, it will break here if screen is resized.

Reply:
Actually... that's fine but I think we need window.screen because this is to detect a mobile device which doesn't change.  There's a different bug here because I should be window.screen.width (I think) butI will fix it.